### PR TITLE
FS-3780 Updating manifest

### PR DESCRIPTION
**Pull request by aaronwilliamsv1**
### Change description
* Adding uat and production values to copilot config

+ref: FS-3780
+semver: minor

- [N/A] Unit tests and other appropriate tests added or updated
- [N/A] README and other documentation has been updated / added (if needed)
- [*] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

File: copilot/fsd-authenticator/manifest.yml [hunk 1]
```diff
@@ -70,20 +70,40 @@ secrets:
   AZURE_AD_TENANT_ID: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/AZURE_AD_TENANT_ID
 
 # You can override any of the values defined above by environment.
-environments:
+environments:  test:
+  deployment:
+    rolling: 'recreate'
   uat:
     variables:
       ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: true
```
Comment (robk-dluhc) [inline copilot/fsd-authenticator/manifest.yml:78]: This was here already, but why is it here for UAT only?  (Probably just I don't understand what it's for)

**Review by robk-dluhc — DISMISSED** [review]

File: copilot/fsd-authenticator/manifest.yml (hunk 1 above)
Comment (aaronwilliamsv1) [reply copilot/fsd-authenticator/manifest.yml:78]: The magic link part?

We did go through this as part of this build - in cloudfoundry, only UAT has this environment variable set

cf env funding-service-design-authenticator-dev
Not set

cf target -s sandbox
cf env funding-service-design-authenticator-test
Not set

cf target -s uat
cf env funding-service-design-authenticator
ACCOUNT_STORE_API_HOST: xxx
ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: true 

cf target -s production
cf env funding-service-design-authenticator
ACCOUNT_STORE_API_HOST: xxx
ALLOW_ASSESSMENT_LOGIN_VIA_MAGIC_LINK: false

Actually, you make a good point - the _default_ is set to true, so this should probably be set to true for both dev and test 🤔

Comment (aaronwilliamsv1) [reply copilot/fsd-authenticator/manifest.yml:78]: https://github.com/communitiesuk/funding-service-design-authenticator/blob/main/config/envs/default.py#L106C5-L108 (default set to true)

**Review by robk-dluhc — APPROVED** [review]

File: copilot/fsd-authenticator/manifest.yml (hunk 1 above)
Comment (aaronwilliamsv1) [reply copilot/fsd-authenticator/manifest.yml:84]: Added in dev and test

**Pull request merged**